### DISCUSSION TITLE
L-BFGS-B

### DISCUSSION THREAD

**Discussion by franz3105** [jax-ml/jax#13007] [Q&A]
Dear all,

how long do you think it will take approximately for the L-BFGS-B code in jax.scipy.optimize to be considered reliable? Or alternatively, if I decide to use it and in case modify it, what major issues should I be aware of?

Thank you in advance.

**Comment by hawkinsp** [comment]
I think the likely outcome is we will remove the LBFGS implementation in JAX itself, instead pointing people to JAXopt https://github.com/google/jaxopt

Check out JAXopt? You might want to pose the same question in their repository.

**Reply by franz3105** [reply]
Ah wonderful I will check it. Thank you.
